consul/consul.nuspec

```diff
@@ -12,7 +12,12 @@
     <packageSourceUrl>https://github.com/cleung2010/chocolatey-packages/tree/master/consul</packageSourceUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>Consul is a tool for service discovery and configuration. Consul is distributed, highly available, and extremely scalable.
-Uses NSSM as the service wrapper. More info at: https://nssm.cc/</description>
+Uses NSSM as the service wrapper. More info at: https://nssm.cc/
+
+#### Package Parameters
```

Comment:
I've added parameters usage info in description because of official recomendation to add it there :) But please feel free to remove. Also added to README.MD